www/js/incident/post-trip-prompt.js

```diff
@@ -212,19 +213,20 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
             return;
         }
         cleanDataIfNecessary(notification, state, data);
-        if($ionicPlatform.is('ios')) {
-          promptReport(notification, state, data).then(function(res) {
-            if(res == true) {
-              Logger.log("About to go to prompt page");
-              displayCompletedTrip(notification, state, data);
-            } else {
-              Logger.log("Skipped incident reporting");
-            }
-          });
-        } else {
-          Logger.log("About to go to prompt page");
-          displayCompletedTrip(notification, state, data);
-        }
+        displayCompletedTrip(notification, state, data);
+        // if($ionicPlatform.is('ios')) {
```

Comment:
this is the code that displays the prompt if you don't close the app when you end the trip. You saw this during my demo last week and said that you don't see it. This is why :)

In that case, there is no visible notification, it is delivered directly to the open app.

Reply:
I have modified this part along with

```js
  var displayCompletedTrip = function(notification, state, data) {
      $rootScope.displayingIncident = true;
      $rootScope.notificationData = notification.data;
      Logger.log("About to display completed trip from Notification");
      $state.go('root.main.diary');
  };
```

to direct notification to display the survey

Reply:
I see. If the user has the app open when the trip ends, it can be a bit disorienting to suddenly have the app move to a different screen with no user input. But I agree that the odds of the user having the app UI open during a trip end is low.